mobile/android/app/src/main/java/com/guruhub/MainActivity.java

```diff
@@ -49,6 +50,7 @@ protected boolean isConcurrentRootEnabled() {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+      RNBootSplash.init(getPlainActivity());
       super.onCreate(null);
     }
```

Comment:
Don't modify onCreate, move new method under it 
```suggestion
    @Override
    protected void onCreate(Bundle savedInstanceState) {
      super.onCreate(null);
    }
    
    @Override
    protected void loadApp(String appKey) {
      RNBootSplash.init(getPlainActivity());
      super.loadApp(appKey);
    }
```